gromacs/protocols/protocol_MD_simulation.py

```diff
@@ -445,7 +445,8 @@ def callMDRun(self, tprFile, saveTrj=True):
         stage = os.path.split(stageDir)[-1]
         gpuStr = ''
         if getattr(self, params.USE_GPU):
-            gpuStr = ' -nb gpu -gpu_id {}'.format(getattr(self, params.GPU_LIST))
+            gpuList = getattr(self, params.GPU_LIST).get().replace(' ', '')
```

Comment:
then shouldn't this have a comma not a space?

Reply:
not really, gromacs accepts the comma separated
the replace would delete hypothetical intemediate spaces ("1, 2" -> "1,2")

Reply:
I know Gromacs would understand "1 and 2" with "12", but I don't like it, you have a twelve there. So I expect the input in the comma separated file, which they also accept and it's more robust

Reply:
ah, ok. I see. 

Does the comma separated work for 1,1,2,2 as well?

Reply:
why would you repeat the 1 and 2?